docs/samplers.txt

```diff
@@ -53,14 +53,21 @@ recommended for direct use by the user, rather it should be accessed via the
 Nested Samplers
 ---------------
 
+Native
+======
+
 - Dynesty: :code:`bilby.core.sampler.dynesty.Dynesty`
 - Nestle :code:`bilby.core.sampler.nestle.Nestle`
 - CPNest :code:`bilby.core.sampler.cpnest.Cpnest`
 - PyMultiNest :code:`bilby.core.sampler.pymultinest.Pymultinest`
-- PyPolyChord :code:`bilby.core.sampler.polychord.PyPolyChord`
 - UltraNest :code:`bilby.core.sampler.ultranest.Ultranest`
-- DNest4 :code:`bilby.core.sampler.dnest4.DNest4`
-- Nessai :code:`bilby.core.sampler.nessai.Nessai`
+
+External
+========
+
+- DNest4 :code:`dnest4-bilby`
+- Nessai :code:`nessai-bilby`
+- PyPolyChord :code:`pypolychord-bilby`
```

Comment:
I realized this is partially duplicates what is here: https://bilby-dev.github.io/bilby/plugins.html#sampler-plugin-library.

It probably makes more sense to have the list where you've added it rather than in the plugins. What do you think?